common/artifacts/901_wc_artifacts_weapon_onehanded.txt

```diff
@@ -78,6 +79,31 @@ ironfoe = {
 
 	slot = weapon
 }
+the_kingslayer = {
+	quality = 4
```

Comment:
I'd put it at tier 3, like Arthas' hammer or Anduin Lothar's sword.

Reply:
I think you right.